### PR TITLE
Add version command

### PR DESCRIPTION
**Pull request by danke-sra**
- Add version command to print the current GVRf version
- Fix error message of some existing commands

### CODE REVIEW

File: GVRf/Framework/src/org/gearvrf/debug/ShellCommandHandler.java [hunk 1]
```diff
@@ -42,25 +43,31 @@ public ShellCommandHandler(GVRContext gvrContext) {
...
         if (engine == null) {
-            mShell.getOutputConverter().convertOutput("Cannot find the language engine for " + language);
-            return;
+            return "Cannot find the language engine for " + language;
         }
 
         mScriptHandler = new ScriptHandler(mGVRContext, language, engine);
         mShell.setLineProcessor(mScriptHandler);
+
+        return null;
```
Comment (thomasflynn) [inline GVRf/Framework/src/org/gearvrf/debug/ShellCommandHandler.java]: you changed the return type to Object, but only return non-null when the engine is null.  correct?  this is just for outputting an error?

Comment (danke-sra) [inline GVRf/Framework/src/org/gearvrf/debug/ShellCommandHandler.java]: Right. I think return type Object is too broad. Maybe String is better?

Comment (thomasflynn) [inline GVRf/Framework/src/org/gearvrf/debug/ShellCommandHandler.java]: that might make things more clear.  i was initially expecting the engine or some other object to be returned.  at first, i thought it was always returning null until i looked at it closer.

**Comment by thomasflynn** [comment]
otherwise, looks good; works as expected.

**Pull request merged**